### DISCUSSION TITLE
Minimum requirements for laptops?

### DISCUSSION THREAD

**Discussion by quetzalsly** [upscayl/upscayl#1048] [Q&A] [answered]
My laptop gpu isnt that good, I was just wondering if this would work somehow? I dont care if its 10x slower. I could just run it overnight.

**Comment by originalbluefox** [comment, marked as answer]
Hello! I've been using upscaly for a year or so, yes you can use it on a laptop afaik. I've done it on my intel uhd 6000 graphics. Its a lot faster on desktop gpus like my 7700xt

**Comment by aaronliu0130** [comment]
We recommend a discrete GPU; it'll probably still run on CPUs if  you do #580, but that'd be terribly slow and is not guaranteed.
